about.html

```diff
@@ -5,67 +5,79 @@
   <meta charset="utf-8">
 
   <title>Sprint Challenge - About</title>
-
+  <link rel="stylesheet" href="https://use.typekit.net/eml1nbi.css">
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
```

Comment:
we should talk about tradeoffs of adding another .css file (about.css)

Reply:
👍🏼. With respect to the Typekit CSS, I would just make a decision and remove either the Typekit or Google Font stylesheet.